### DISCUSSION TITLE
솔루션 심사 완료 후 솔루션 등록정보 변경 가능여부 문의

### DISCUSSION THREAD

**Discussion by githubcnsbs** [commerce-api-naver/commerce-solutions#26] [묻고 답하기] [answered]
안녕하세요. 

솔루션 심사 완료 후 솔루션 등록 정보를 변경할 수 있나요.

예) 
1. 솔루션 CBT를 [바로 사용형]으로 진행 후 [심사형 승인형]으로 변경
2. 이벤트 훅 사용여부 안함 > 사용함으로 변경
3. API 그룹 선택 미사용 > 사용  혹은 API 추가

가능하다면 변경 프로세스 가이드가 있을까요.

**Comment by commerce-api-naver** [comment, marked as answer]
안녕하세요. 커머스API 이민석입니다.

솔루션의 심사 완료 후 일부 정보에 대해서만 변경이 가능합니다.
'이벤트 훅' 관련 정보와 같이 심사된 내용에 영향이 없으면서 솔루션의 기술적 운용과 관련한 변경은 개발사가 직접 변경할 수 있습니다.

CBT 기간 동안의 솔루션 변경 혹은 심사 정책에 대한 확인이 필요하시다면 개발사명과 구체적으로 확인이 필요한 내용을 알려주시면 추가 확인을 도와드리겠습니다.

**Reply by githubcnsbs** [reply]
네, 답변 감사드립니다.
